### DISCUSSION TITLE
Custom CSS

### DISCUSSION THREAD

**Discussion by danihazler** [onweru/hugo-swift-theme#72] [Q&A]
Hi there,

What would be the best way to implement custom CSS within your theme?

I've followed some solutions on this thread https://discourse.gohugo.io/t/how-to-override-css-classes-with-hugo/3033/3
but I keep getting a console error
```
Refused to apply style from 'http://localhost:1313/static/sass/index.sass' because its MIME type ('text/plain') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
```

**Comment by onweru** [comment]
It seems like you’re adding a `.sass` file.

**Reply by onweru** [reply]
Instead of a `.css` file

**Reply by danihazler** [reply]
Actually, the problem was that I was adding the custom styles in the wrong folder... I was adding inside `static` and it should be within the root.

```
.
├── archetypes
├── assets   // --> here
├── content
├── data
├── layouts
├── static // --> not here
└── themes
```

btw, thanks a lot for the theme! :)

**Reply by onweru** [reply]
Awesome. I'm glad you like it.
